src/reducers/activities.js

```diff
@@ -0,0 +1,26 @@
+const activitiesReducerDefaultState = [];
```

Comment:
Imo state shuould be an object:
```
{
 activities: []
}
```
Probably you will have more fields here in this object.

Reply:
jesus christ, it sounds like a ton of corrections and modifications, are you sure? :(

Reply:
yes.

Reply:
but you can manage it by separate PR